spec/models/global_survey_response_map_spec.rb

```
@@ -22,7 +22,7 @@
     describe '#get_title' do
       it 'returns the correct title constant' do
         global_survey_response_map = GlobalSurveyResponseMap.new
-        expect(global_survey_response_map.get_title).to eq(ResponseMapSubclassTitles::GLOBAL_SURVEY_RESPONSE_MAP_TITLE)
+        expect(global_survey_response_map.get_title).to eq(ExpertizaConstants::ResponseMapTitles::GLOBAL_SURVEY_RESPONSE_MAP_TITLE)
       end
     end
-  end
\ No newline at end of file
+  end
```

RuboCop (1.86.1)

[convention] `end` at 28, 2 is not aligned with `RSpec.describe GlobalSurveyResponseMap, type: :model do` at 3, 0.

(Layout/BlockAlignment)